### PR TITLE
[ MNIST ] Add save weight initialization data

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR includes codes to save weight initialized data for nntrainer
input. This can be read using NN.readModel() except optimizer read in
fully connected layer.

Signed-off-by: jijoongmoon <jijoong.moon@samsung.com>

### CODE REVIEW

File: Applications/mnist/Tensorflow/Training_Keras.py [hunk 1]
```diff
@@ -41,15 +41,12 @@
...
+num_epoch = 1500
 DEBUG = True
 USE_FIT = False
 
 def save(filename, *data):
-    if os.path.isfile(filename):
-        os.remove(filename)
-
-    with open(filename, 'ab') as outfile:
+    with open(filename, 'a+') as outfile:
```
Comment (zhoonit) [inline Applications/mnist/Tensorflow/Training_Keras.py:49]:
```suggestion
    with open(filename, 'ab+') as outfile:
```

File: test/input_gen/genInput.py [hunk 2]
```diff
@@ -51,6 +51,12 @@ def save(filename, *data):
 def gen_tensor(shape, dtype=dtypes.float32):
   return np.random.random_sample(shape)
 
+##
+# @brief generate random tensor
+def gen_tensor_int(shape):
+  rand_t = np.random.randint(1, 100, size=shape)
```
Comment (kparichay) [inline test/input_gen/genInput.py:57]: lets make it (1, 10)
1, 100 causes overflow sometimes

Comment (jijoongmoon) [reply test/input_gen/genInput.py:57]: This change is related with another PR #494

File: Applications/mnist/Tensorflow/Training_Keras.py (hunk 1 above)
Comment (jijoongmoon) [reply Applications/mnist/Tensorflow/Training_Keras.py]: fixed. thanks.

**Review by zhoonit — APPROVED** [review]
LGTM

**Review by kparichay — COMMENTED** [review]
LGTM

**Review by kparichay — APPROVED** [review]

**Pull request merged**